lib/modules/session.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const { v4: uuid } = require('uuid');
+
+const newSession = (websocket, params) => {
+  // TODO: match requested capabilities
+  // const { capabilities } = params;
+  websocket.sessionId = uuid();
+  return {
+    sessionId: websocket.sessionId,
+    capabilities: {
+      atName: 'TODO',
```

Comment:
Not sure how much we want to do with capabilities here - but the harness does use this information so @mzgoddard will want it to be accurate

Reply:
Yeah for `atName` and `atVersion` we'll have to detect the running AT software. That will probably be specific to each AT application. Like for `atName` we can probably scan running processes for certain names. But `atVersion` will probably then be specific to each AT.

We should be able to do `platformName` now with [`os.platform()`](https://nodejs.org/docs/latest/api/os.html#osplatform)?